### ISSUE TITLE
Possibly unsuitable parsing format for semester-wise grades

### ISSUE THREAD

**Issue by VinayGupta23** [aneesh-neelam/VITacademics#78]
In the Json string for grades, `"semester_wise"` object in turn contains multiple child objects, whereas an array would be more suited since it is a list of semesters.
Find below a sample of the Json, along with suggested change:

Current:

``` Json
"semester_wise": {
        "2013-11": {
            "credits": 21,
            "gpa": "8.28"
        },
        "2014-05": {
            "credits": 21,
            "gpa": "8.57"
        },
        "2014-11": {
            "credits": 24,
            "gpa": "8.46"
        },
        "2014-07": {
            "credits": 8,
            "gpa": "8.00"
        }
    }
```

Suggested change:

``` Json
"semester_wise": [
        {
            "exam_held": "2013-11",
            "credits": 21,
            "gpa": "8.28"
        },
        {
            "exam_held": "2014-05",
            "credits": 21,
            "gpa": "8.57"
        },
        {
            "exam_held": "2014-11",
            "credits": 24,
            "gpa": "8.46"
        },
        {
            "exam_held": "2014-07",
            "credits": 8,
            "gpa": "8.00"
        }
    ]
```

**Comment by karthikb351** [comment]
+1 this.

Makes no sense to have the additional nesting.

**Comment by aneesh-neelam** [comment]
@VinayGupta23: Will convert to an array.

**Comment by aneesh-neelam** [comment]
Fixed in 22286ab51f9e2cd5fd76b09285831d375d3b99e9.
